frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-builder/task-sidebar/temp-utils.tsx

```diff
@@ -30,8 +30,10 @@ export const StringParam: React.FC<ParameterProps> = (props) => {
   const { currentValue, defaultValue, isValid, name, onChange, setDirty } = props;
 
   return (
-    <TextInput
+    <TextArea
       id={name}
+      rows={1}
+      style={{ minHeight: 30, resize: 'vertical' }}
```

Comment:
My other comment is more-or-less for posterity if we need to come back to this.

This comment however is a request for a change.

```suggestion
      style={{ minHeight: 30 }}
      resizeOrientation="vertical"
```
There is a prop for the resize style. Best to use that.